site/jms-client.md

```diff
@@ -367,6 +367,164 @@ may have configuration files or command-line options.
 Refer to the documentation for the target logging framework
 for configuration details.
 
+## <a id="rpc-support" class="anchor" href="#rpc-support">Support for Request/Reply (a.k.a. RPC)</a>
+
+It is possible to use JMS for synchronous request/reply use cases.
+This pattern is commonly known as *Remote Procedure Call* or *RPC*.
+
+### With JMS API
+
+An RPC client can be implemented in pure JMS like the following:
+
+<pre class="sourcecode java">
+Message request = ... // create the request message
+// set up reply-to queue and start listening on it
+Destination replyQueue = session.createTemporaryQueue();
```

Comment:
It would be worth noting that in a more typical use-case, you would persist the reply-to queue to avoid the cost of setup/teardown on each well.

Reply:
OK, mentioned this solution as welll.